frontend/src/pages/voting/index.tsx

```diff
@@ -104,6 +104,10 @@ export const VotingPage = () => {
     ) !== -1;
 
   const isVotingClosed = data && !data.conference.isVotingOpen;
+  const userCannotVote =
+    loggedIn && (loading || (cannotVoteErrors ?? false) || (error ?? false));
+  const showFilters =
+    isVotingClosed !== undefined && !isVotingClosed && !userCannotVote;
```

Comment:
if we make `isVotingClosed` always a boolean this can just be

```suggestion
  const showFilters = !isVotingClosed && !userCannotVote;
```

Reply:
seems working :)